docs/using-docker.rst

```diff
@@ -72,5 +70,6 @@ Running with Compose
 
 Run these commands to begin using Teleirc with Docker Compose.
 
-#. Copy ``containers/docker-compose.yml.example`` to ``docker-compose.yml``
+#. Copy ``containers/docker-compose.yml.example`` to ``containers/docker-compose.yml``
+#. Copy ``.env.example`` to ``containers/.env`` and edit accordingly.
```

Comment:
Maybe I missed something here – will the containers still build if the user does not move these files to the root level of the repository? I didn't think docker-compose would work as expected from the `containers/` directory.

Reply:
It won't run and will error out. The path is relative to where the compose file is located.